pyaedt/application/Analysis.py

```diff
@@ -63,16 +64,12 @@ def __init__(self, application, projectname, designname, solution_type, setup_na
 
     @property
     def materials(self):
-        """Material Manager that is used to manage materials in the project.
+        """Material manager for managing materials in the project.
```

Comment:
```suggestion
        """Manages materials in the project.
```
"Material manager for managing materials" sounded a bit wordy to me. Feel free to ignore this suggestion.

Reply:
@jleonatti Totally agree. I just wasn't sure who wrote the original and didn't want to offend them!